build/run-tests.sh

```diff
@@ -1,7 +1,25 @@
 #!/usr/bin/env bash
+set -e
 
-docker-compose build --no-cache
+function stop_containers() {
+  docker-compose down
+}
+
+trap stop_containers EXIT
+
+if [[ ! -d ../verify-event-system-database-scripts ]]; then
+    echo "Cloning database migration scripts repo..."
+    pushd ..
+    git clone git@github.com:alphagov/verify-event-system-database-scripts.git
+    popd
+fi
+
+echo "Building test imgages..."
+docker-compose build
```

Comment:
Are we going to need to suppress caching again here?

Reply:
Not anymore ... the only reason that was previously needed was due to the way we previously built a custom Postgres image that used a `RUN` command to pull the scripts repo, which meant layer caching wouldn't work. I've deliberately removed the `--no-cache` as,  I believe, we won't fall prey to that particular problem any more and we can take advantage of layer caching again :-)

Reply:
Ahh yes. Thank you for the reminder. 👍